scene/main/node.h

```diff
@@ -507,6 +510,15 @@ class Node : public Object {
 	void set_unique_name_in_owner(bool p_enabled);
 	bool is_unique_name_in_owner() const;
 
+	void set_exposed_in_owner(bool p_enabled);
```

Comment:
Do we need these methods and information when not running tools build?

Reply:
I believe for a similar reason as set_unique_name_in_owner and set_editable_instance.  The nodes still need these flags in order for the scene to reconstruct it.  Could be wrong here, but in testing, moving these to #ifdef TOOLS_ENABLED causes scenes to fail to load.